### DISCUSSION TITLE
Cann't save user defined style

### DISCUSSION THREAD

**Discussion by 9Roman9** [freeplane/freeplane#332] [Help] [answered]
Hi everyone! 
I faced the next problem. 

I create new style and save it. It can be used in current map. However,  it vanished in new map or after restart the program so it's impossible to use this style repeatedly in other map.

Are there ways to fix this problem?

Thanks in advance for any insight you might have.

**Comment by macmarrum** [comment, marked as answer]
Hi @9Roman9 

For a style to be available in other mindmaps, it needs to be saved to a [template](https://freeplane.github.io/wiki/#/templates).
Once this is done, the style will be available in each new mindmap created from that template.

There are 2 ways to do it
1. Open the template mindmap and add your new style there.
2. Add a new style to your regular mindmap.
In `Tool panel` → `Make current formatting default`, after setting the properties of your new style, choose `for all mindmaps based on template:` and click `Apply`. 
Use `Choose template` to change the association beforehand, if needed.

**Reply by 9Roman9** [reply]
It works. Thank you!
